prog/weaveutil/datapath.go

```diff
@@ -3,16 +3,36 @@ package main
 
 import (
 	"fmt"
+	"net"
 	"os"
+	"strconv"
+	"syscall"
 
 	"github.com/weaveworks/weave/common/odp"
+	wnet "github.com/weaveworks/weave/net"
 )
 
 func createDatapath(args []string) error {
-	if len(args) != 1 {
-		cmdUsage("create-datapath", "<datapath>")
+	var (
+		mtu    int
+		err    error
+		setMTU bool
+	)
+
+	if len(args) != 1 && len(args) != 3 {
+		cmdUsage("create-datapath", "[--mtu <mtu>] <datapath>")
```

Comment:
`create-datapath` is used in two places in the weave script. Here https://github.com/weaveworks/weave/blob/7c51c313579d11c92d1840c365a4dddf7aad6e15/weave#L628 you probably don't want to mess with setting mtu and doing the check.